tutorials/Qa_skullstrip.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial for QA of Skull Strip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is designed to illustrate the QA code used to generate the figure for the stripped brain and skull."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "this code will do 3 main things:\n",
+    "\n",
+    "1. Input original t1w file and the skull-striped brain file\n",
+    "2. Shows the skull-stripped brain (green) overlaid on the original t1w (magenta)\n",
+    "3. save the image into output directory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So that we can open the QA image to see if the skull strip result is good or not."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "#import packages\n",
+    "import warnings\n",
+    "\n",
+    "warnings.simplefilter(\"ignore\")\n",
+    "import sys\n",
+    "import nibabel as nib\n",
+    "import numpy as np\n",
+    "import os\n",
+    "from PIL import Image, ImageDraw,ImageFont\n",
+    "import matplotlib.pyplot as plt\n",
+    "from ndmg.stats.qa_skullstrip import gen_overlay_pngs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## inputs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "original is the path to the original t1w brain, with the skull included\n",
+    "\n",
+    "outdir is the path to the directory where QA image will be saved\n",
+    "\n",
+    "you can change it to your own path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "original = r'/mnt/f/JHU/ndd/dataset/part_of_SWU4/sub-0025629_ses-1_T1w.nii.gz'\n",
+    "outdir  = r'/mnt/f/JHU/ndd/dataset/output1/sub-0025864/ses-1/'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run AFNI 3dSkullStrip to do skull strip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "use AFNI 3dSkullStrip to do skull strip for the original t1w brain"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "brainfile = f\"{outdir}only_brain.nii.gz\"\n",
+    "cmd = f\"3dSkullStrip -prefix {brainfile} -input {original} -ld 30\"\n",
+    "os.system(cmd)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## run qa_skullstrip.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "call the function gen_overlay_pngs"
```

Comment:
Add what each of the input values means (what does minthr mean? what does edge mean? etc.)

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/65e3450758390ab8ed42ce8c64b32d8bacb71ae9